apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/AplDbVersion.java

```diff
@@ -688,38 +688,52 @@ protected int update(int nextUpdate) {
                 apply("CREATE INDEX IF NOT EXISTS genesis_public_key_height_idx on genesis_public_key(height)");
             case 251:
                 // SHARDING meta-info inside main database
-                apply("CREATE TABLE IF NOT EXISTS shard (shard_id BIGINT AUTO_INCREMENT NOT NULL, shard_hash VARBINARY not null, shard_state BIGINT)");
+                apply("CREATE TABLE IF NOT EXISTS shard (shard_id BIGINT AUTO_INCREMENT NOT NULL, shard_hash VARBINARY, " +
+                        "shard_height BIGINT not null, shard_state BIGINT)");
```

Comment:
height should be int

Reply:
why should ? Why we can have it as Long for limited number of rows?

Reply:
Look for height type in other tables and you will understand. Long type is misleading

Reply:
Replaced by Int - Integer